src/ui/components/Icon/styles.scss

```diff
@@ -44,6 +44,16 @@
   width: 112px;
 }
 
+.Icon-triangle-down {
+  background: url("./triangle-down.svg") center no-repeat;
+  background-size: contain;
+}
+
+.Icon-triangle-down-black {
+  background: url("./triangle-down-black.svg") center no-repeat;
```

Comment:
I checked out this branch locally and I don't see an arrow icon in the header (in mobile view). Maybe it's because you need to use the new `icon` mixin?